cpp/README.md

```diff
@@ -1,118 +1,119 @@
 # Ice for C++
 
-[Getting started] | [Examples] | [NuGet packages] | [Documentation] | [Building from source]
+[Examples] | [Ice Manual] | [Building from source]
 
 The [Ice framework] provides everything you need to build networked applications,
 including RPC, pub/sub, server deployment, and more.
 
-Ice for C++ is the C++ implementation of Ice.
+Ice for C++ is the C++ implementation of the Ice framework.
 
 ## Sample Code
 
 ```slice
-// Slice definitions (Hello.ice)
+// Slice definitions (Greeter.ice)
 
 #pragma once
 
-module Demo
+module VisitorCenter
 {
-    interface Hello
+    /// Represents a simple greeter.
+    interface Greeter
     {
-        void sayHello();
+        /// Creates a personalized greeting.
+        /// @param name The name of the person to greet.
+        /// @return The greeting.
+        string greet(string name);
     }
 }
 ```
 
 ```cpp
 // Client application (Client.cpp)
 
+#include "Greeter.h"
+
 #include <Ice/Ice.h>
-#include <Hello.h>
+#include <iostream>
 
 using namespace std;
-using namespace Demo;
 
-int
-main(int argc, char* argv[])
+int main(int argc, char* argv[])
 {
-    try
-    {
-        const Ice::CommunicatorHolder ich(argc, argv);
-        HelloPrx hello{ich.communicator(), "hello:default -h localhost -p 10000"};
-        hello->sayHello();
-    }
-    catch (const std::exception& ex)
-    {
-        cerr << ex.what() << endl;
-        return 1;
-    }
+    Ice::CommunicatorPtr communicator = Ice::initialize(argc, argv);
+    Ice::CommunicatorHolder communicatorHolder{communicator};
+
+    VisitorCenter::GreeterPrx greeter{communicator, "greeter:tcp -h localhost -p 4061"};
+
+    string greeting = greeter.greet("alice");
```

Comment:
Should we prefer `->` like in the demos?

Reply:
Yes, even though I find this syntax a bit odd without a comment.

What's the advantage of `->`?
 - it's the traditional invocation syntax in C++ (so highly desirable to keep it)
 - it allows us to use the same invocation syntax with non-nullable proxies and nullable proxies (thanks to `operator->`  on std::optional).

Will fix.